db/migrate/20190606134257_change_usages_of_old_accounts_to_match_nomenclature.rb

```diff
@@ -0,0 +1,26 @@
+class ChangeUsagesOfOldAccountsToMatchNomenclature < ActiveRecord::Migration
+  CHANGED = { fixed_asset_depreciations: :financial_asset_depreciations,
+              fixed_assets: :financial_assets,
+              fixed_assets_suppliers: :financial_assets_suppliers ,
+              fixed_assets_values: :financial_assets_values }
+              
+  def up
+    CHANGED.each do |to, from|
+      execute <<-SQL
+        UPDATE accounts
+           SET usages = '#{to}'
+         WHERE usages = '#{from}'
```

Comment:
the `usages` name seems to indicate that its possible to have multiple usages. @Aquaj ?